SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/VisibilityTest/ReplicatedVisibilityTestActor.h

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "CoreMinimal.h"
+#include "SpatialGDKFunctionalTests/SpatialGDK/TestActors/ReplicatedTestActorBase.h"
+#include "ReplicatedVisibilityTestActor.generated.h"
+
+UCLASS()
+class AReplicatedVisibilityTestActor : public AReplicatedTestActorBase
+{
+	GENERATED_BODY()
+
+public:
+	AReplicatedVisibilityTestActor();
+
+	void GetLifetimeReplicatedProps(TArray<FLifetimeProperty>& OutLifetimeProps) const override;
```

Comment:
I think you might not need this anymore, together with its implementation in the ReplicatedVisibilityTestActor.cpp, since it seems like no properties are replicated on this Actor?